### PR TITLE
changed the readme files in 4.0-dev

### PR DESCRIPTION
**Pull request by jagatheeswar**
Pull Request for Issue # .

### Summary of Changes

just added 2 words. just for a practice.

### Testing Instructions
no need to test.

### Actual result BEFORE applying this Pull Request
Ready to install Joomla? both in readme.md and readme.txt

### Expected result AFTER applying this Pull Request

Are you ready to install Joomla? both in readme.md and readme.txt

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,65 +3,78 @@ Joomla! CMS™ [![Analytics](https://ga-beacon.appspot.com/UA-544070-3/joomla-cm
...
-* Joomla! 4.0 [version history](https://docs.joomla.org/Special:MyLanguage/Joomla_4.0_version_history).
-* Detailed changes are in the [changelog](https://github.com/joomla/joomla-cms/commits/4.0-dev).
+* Joomla! 3.9 [version history](https://docs.joomla.org/Special:MyLanguage/Joomla_3.9_version_history).
+* Detailed changes are in the [changelog](https://github.com/joomla/joomla-cms/commits/staging).
 
 What is Joomla?
 ---------------------
 * [Joomla!](https://www.joomla.org/about-joomla.html) is a **Content Management System** (CMS) which enables you to build websites and powerful online applications.
-* It is a simple and powerful web server application which requires a server with PHP and either MySQL or PostgreSQL to run. You can find [full technical requirements here](https://downloads.joomla.org/technical-requirements).
+* It is a simple and powerful web server application which requires a server with PHP and either MySQL, PostgreSQL or SQL Server to run. You can find [full technical requirements here](https://downloads.joomla.org/technical-requirements).
```
Comment (ReLater) [inline README.md:20]: Joomla 4 doesn't support "SQL Server".

File: README.md [hunk 2]
```diff
@@ -3,65 +3,78 @@ Joomla! CMS™ [![Analytics](https://ga-beacon.appspot.com/UA-544070-3/joomla-cm
...
 
-Overview
+What is this?
 ---------------------
-* This is the source of Joomla! 4.x.
+* This is a Joomla! 3.x installation/upgrade package.
 * Joomla's [Official website](https://www.joomla.org).
-* Joomla! 4.0 [version history](https://docs.joomla.org/Special:MyLanguage/Joomla_4.0_version_history).
-* Detailed changes are in the [changelog](https://github.com/joomla/joomla-cms/commits/4.0-dev).
+* Joomla! 3.9 [version history](https://docs.joomla.org/Special:MyLanguage/Joomla_3.9_version_history).
```
Comment (ReLater) [inline README.md:14]: If it's a readme for Joomla 4 why did you change this to Joomla 3.9?

File: README.md [hunk 3]
```diff
@@ -3,65 +3,78 @@ Joomla! CMS™ [![Analytics](https://ga-beacon.appspot.com/UA-544070-3/joomla-cm
...
-| [![Build Status](https://ci.joomla.org/api/badges/joomla/joomla-cms/status.svg?branch=4.0-dev)](https://ci.joomla.org/joomla/joomla-cms) | [![Build status](https://ci.appveyor.com/api/projects/status/ru6sxal8jmfckvjc/branch/4.0-dev?svg=true)](https://ci.appveyor.com/project/release-joomla/joomla-cms) | [![PHP](https://img.shields.io/badge/PHP-V7.2.5-green)](https://www.php.net/) | [![node-lts](https://img.shields.io/badge/Node-V12.0-green)](https://nodejs.org/en/) | [![npm](https://img.shields.io/badge/npm-v6.13.4-green)](https://nodejs.org/en/) |
+| Drone-CI | AppVeyor |
+| ------------- | ------------- |
+|  [![Build Status](https://ci.joomla.org/api/badges/joomla/joomla-cms/status.svg)](https://ci.joomla.org/joomla/joomla-cms)  | [![Build status](https://ci.appveyor.com/api/projects/status/ru6sxal8jmfckvjc/branch/staging?svg=true)](https://ci.appveyor.com/project/release-joomla/joomla-cms)  |
 
-Overview
+What is this?
 ---------------------
-* This is the source of Joomla! 4.x.
+* This is a Joomla! 3.x installation/upgrade package.
```
Comment (ReLater) [inline README.md:12]: If it's a readme for Joomla 4 why did you change this to Joomla 3.x?

File: README.md [hunk 4]
```diff
@@ -3,65 +3,78 @@ Joomla! CMS™ [![Analytics](https://ga-beacon.appspot.com/UA-544070-3/joomla-cm
...
 * Joomla's [Official website](https://www.joomla.org).
-* Joomla! 4.0 [version history](https://docs.joomla.org/Special:MyLanguage/Joomla_4.0_version_history).
-* Detailed changes are in the [changelog](https://github.com/joomla/joomla-cms/commits/4.0-dev).
+* Joomla! 3.9 [version history](https://docs.joomla.org/Special:MyLanguage/Joomla_3.9_version_history).
+* Detailed changes are in the [changelog](https://github.com/joomla/joomla-cms/commits/staging).
```
Comment (ReLater) [inline README.md:15]: I'm not sure that Joomla 4 developement will go into branch "staging" as long as Joomla 3 will be supported by the community. Joomla 3 is located in branch "staging".

**Comment by richard67** [comment]
Closing as not a real PR:

> just added 2 words. just for a practice.

@jagatheeswar If you want to get practice with handling pull requests, you can create an own fork of this repository (if not already done) and then you can create branches and make pull requests in your own repository.

This will not annoy other contributors and maintainers of this repository here, who need their time to fix real bugs and issues.

Thank you for your understanding.
